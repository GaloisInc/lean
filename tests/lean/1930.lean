structure S  := (f : â„•)

def F : S := { f := prod.1 }
