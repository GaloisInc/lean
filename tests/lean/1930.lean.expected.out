1930.lean:3:24: error: invalid field notation, type is not of the form (C ...) where C is a constant
  prod
has type
  Type ? → Type ? → Type (max ? ?)
